calplus/conf/providers.py

```diff
@@ -29,23 +29,25 @@
                default=''),
     cfg.StrOpt('os_novaclient_version',
                default='2.1'),
+    cfg.StrOpt('os_swiftclient_version',
+               default='2'),
```

Comment:
which OpenStack version you want to use? Newton is 3, so I think you want to use Mitaka, right?
https://releases.openstack.org/newton/#newton-python-swiftclient